### PR TITLE
BDS-59 Add notification component

### PR DESCRIPTION
**Pull request by JonasNiestroj**

### CODE REVIEW

File: css/components/notification.postcss [hunk 1]
```diff
@@ -3,14 +3,55 @@
 \*----------------------------------------*/
 
 .notification {
-  @apply flex items-start;
+  @apply flex items-center;
+  @apply max-w-lg;
   @apply p-4 rounded;
+  @apply text-sm;
 
   &.is-error {
```
Comment (sylvainaerni) [inline css/components/notification.postcss:11]: I see two options here to keep consistency with the css names:
1. like the other existing components we write `.notification .notification--name-of-the-modifier` (check badge.postcss file for example)
2. we keep `.is-success` and similar but we make them generic

File: css/components/notification.postcss [hunk 2]
```diff
@@ -3,14 +3,55 @@
 \*----------------------------------------*/
 
 .notification {
-  @apply flex items-start;
+  @apply flex items-center;
+  @apply max-w-lg;
   @apply p-4 rounded;
+  @apply text-sm;
```
Comment (sylvainaerni) [inline css/components/notification.postcss:10]: please add a `shadow-lg` here

**Review by sylvainaerni — COMMENTED** [review]
Hi @JonasNiestroj , ready to discuss quickly these propositions here

File: css/components/notification.postcss (hunk 1 above)
Comment (JonasNiestroj) [reply css/components/notification.postcss:11]: I would say lets stick to notification--name-of-the-modifier for the moment. Its more clear and maybe there will be cases where we need to overwrite the normal behavior. WDYT?

**Pull request merged**